<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GenDB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Detail info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Firstname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="surnameLabel">
             <property name="text">
              <string>Surname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="birthLabel">
             <property name="text">
              <string>Birthdate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="deathLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Date of death</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="hwLabel">
             <property name="text">
              <string>Husband/ wife</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="hw2Label">
             <property name="text">
              <string>2. Husband/ wife</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="motherLabel">
             <property name="text">
              <string>Mother</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="fatherLabel">
             <property name="text">
              <string>Father</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="nameEdit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="surnameEdit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="birthEdit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="deathEdit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="hwEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="hw2Edit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="parent1Edit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="parent2Edit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>View</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>3</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="PeopleView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>2</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLineEdit" name="sexSearchLine">
               <property name="statusTip">
                <string>Filter by gender</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nameSearchLine">
               <property name="statusTip">
                <string>Filter by name</string>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="surnameSearchLine">
               <property name="statusTip">
                <string>Filter by surname</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="birthSearchLine">
               <property name="statusTip">
                <string>Filter by birthdate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="deathSearchLine">
               <property name="statusTip">
                <string>Filter by date of death</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QTableView" name="personView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>2</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QPushButton" name="removeRowButton">
               <property name="text">
                <string>Remove row</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="insertRowButton">
               <property name="text">
                <string>Insert row</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Familyview">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>2</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLineEdit" name="famSearchLine">
               <property name="statusTip">
                <string>Filter by family name</string>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="husSearchLine">
               <property name="statusTip">
                <string>Filter by husband name</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="wifeSearchLine">
               <property name="statusTip">
                <string>Filter by wife name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="marrSearchLine">
               <property name="statusTip">
                <string>Filter by marriage date</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="divSearchLine">
               <property name="statusTip">
                <string>Filter by divorce date</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QTableView" name="familyView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>2</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QPushButton" name="removeRowButton_2">
               <property name="text">
                <string>Remove row</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="insertRowButton_2">
               <property name="text">
                <string>Insert row</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="title">
       <string>Statistics</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="textLabel">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Number of people</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="recNum">
             <property name="maximumSize">
              <size>
               <width>600</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Number of families</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="famNum">
             <property name="maximumSize">
              <size>
               <width>600</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="textLabel2">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Source</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sourceStat">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="saveAction"/>
    <addaction name="revertAction"/>
    <addaction name="separator"/>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="closeViewAction"/>
    <addaction name="importAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOpen_MyHeritage"/>
    <addaction name="actionOpen_FamilySearch"/>
    <addaction name="actionOpen_soa_praha_ebadatelna"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newAction">
   <property name="text">
    <string>New database</string>
   </property>
   <property name="toolTip">
    <string>Create new database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>Open database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="importAction">
   <property name="text">
    <string>Import database</string>
   </property>
  </action>
  <action name="revertAction">
   <property name="text">
    <string>Discard changes</string>
   </property>
   <property name="toolTip">
    <string>Delete database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Save changes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="closeViewAction">
   <property name="text">
    <string>Close database</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen_MyHeritage">
   <property name="text">
    <string>Open MyHeritage</string>
   </property>
  </action>
  <action name="actionOpen_FamilySearch">
   <property name="text">
    <string>Open FamilySearch</string>
   </property>
  </action>
  <action name="actionOpen_soa_praha_ebadatelna">
   <property name="text">
    <string>Open ebadatelna.soapraha</string>
   </property>
   <property name="toolTip">
    <string>Open ebadatelna.soapraha</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
